charts/spinnaker/templates/configmap/halyard-config.yaml

```diff
@@ -90,10 +90,10 @@ data:
     {{ end }}
 
     # Docker Registry
-    $HAL_COMMAND config provider docker-registry enable
+    $HAL_COMMAND config provider docker-registry enable {{ if (include "skipDockerRegistryEnablingValidation" .) -}} --no-validate {{- end }}
     {{- range $index, $registry := .Values.dockerRegistries }}
 
-    if $HAL_COMMAND config provider docker-registry account get {{ $registry.name }}; then
+    if $HAL_COMMAND config provider docker-registry account get {{ $registry.name }} {{- if $registry.noValidate }} --no-validate {{- end }}; then
```

Comment:
Do we also need to chomp on the right here like above? (Also on the left above?) 🤔

Reply:
In this one I purposefully chomped left, not right (see last commit). Otherwise, if the value is `false`, it would cause a space to be present between the end of the command and `;`. Doing both would not work as there needs to be at least one space if the value is `true`. 

For completeness we can also do it in the other space. Currently that just leaves a whitespace at the end of the line which isn't as visible (or maybe is auto chomped? Didn't check). But I believe (can't check; on phone) that is a bit different the rest of the code style.

Feel free to change, otherwise I'll take a look tomorrow.

Reply:
Right. 👍